go/teams/handler.go

```diff
@@ -258,8 +258,14 @@ func handleChangeSingle(ctx context.Context, g *libkb.GlobalContext, row keybase
 	defer mctx.Trace(fmt.Sprintf("team.handleChangeSingle(%+v, %+v)", row, change),
 		func() error { return err })()
 
-	HintLatestSeqno(mctx, row.Id, row.LatestSeqno)
-	HintLatestHiddenSeqno(mctx, row.Id, row.LatestHiddenSeqno)
+	err = HintLatestSeqno(mctx, row.Id, row.LatestSeqno)
+	if err != nil {
+		return err
```

Comment:
Unsure if a failure here should exit the function without continuing

Reply:
@heronhaye You tell me!  I have no clue what this is.  Should I just be logging and ignoring it?

Reply:
I think Max or Miles would know better than me

Reply:
@maxtaco @mlsteele thoughts on what to do about an error from `HintLatestSeqno`?  Ignore/log or return it?

Reply:
Ignore would be good